### PR TITLE
Add more queries with transactions and aggregates

### PR DESCRIPTION
**Pull request by Abe1able**
### In this milestone of the project, I have

- used aggregates aggregate functions and GROUP BY to answer analytical questions.
- used transactions to update and delete records.
- used more queries.

#### Screenshot of queries and operations:

Queries2
Queries3
Queries4
Queries5
Queries6

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
  date_of_birth DATE,
  escape_attempts INT,
  neutered BOOLEAN,
- weight_kg NUMERIC
+ weight_kg NUMERIC,
+ species TEXT
```
Comment (mckent05) [inline schema.sql:10]: - You seem to have manually edited your animal's table, I suggest you use the line ` ALTER TABLE animals ADD species varchar(255)` to dynamically update the table

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Abe1able, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job attaching the correct results of screenshots for your queries 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Abe1able) [reply schema.sql:10]: Fixed, thank you.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Abe1able 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**